analyzers/tests/SonarAnalyzer.Test/Rules/AspNet/UseAspNetModelBindingTest.cs

```diff
@@ -0,0 +1,292 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2024 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using SonarAnalyzer.Rules.CSharp;
+
+namespace SonarAnalyzer.Test.Rules;
+
+#if NET
+
+[TestClass]
+public class UseAspNetModelBindingTest
+{
+    private readonly VerifierBuilder builderAspNetCore = new VerifierBuilder<UseAspNetModelBinding>()
+        .WithBasePath("AspNet")
+        .WithOptions(ParseOptionsHelper.FromCSharp12)
```

Comment:
C# 12 isn't really needed for the test cases. You can remove it if you:
- add set the [UseAspNetModelBinding_NonCompliantAccess](https://github.com/SonarSource/sonar-dotnet/blob/3b36922936dd7bce022bf20ca429190cc644746f/analyzers/tests/SonarAnalyzer.Test/Rules/AspNet/UseAspNetModelBindingTest.cs#L55) test case to use C# 11 (because of the raw string literals)
- and instead of `default` return `View()` or `null` in [UseAspNetModelBinding_AspNetCore.cs](https://github.com/SonarSource/sonar-dotnet/blob/3b36922936dd7bce022bf20ca429190cc644746f/analyzers/tests/SonarAnalyzer.Test/TestCases/AspNet/UseAspNetModelBinding_AspNetCore.cs#L52)

Reply:
With the new merged test cases guide line, there are only two test files to create
* SomeTest.cs and
* SomeTest_Latest.cs

I opted to rename UseAspNetModelBinding_AspNetCore.cs to UseAspNetModelBinding_AspNetCore_Latest.cs and use LangVersion.Latest because for .Net Core the recommendation is to use latest. Is this okay for you?